docs/api/wanchain.md

```diff
@@ -1,5 +1,255 @@
 # API / Wanchain
 
+## Deploy gateway
+
+Deploy gateway allows for delegating deployment of an `AssetLedger` smart contract to a third party and transfering value in the same atomic transaction.
```

Comment:
Deploy gateway allows for delegating deployment of an `AssetLedger` smart contract to a third party and for transferring value in the same atomic transaction.